Source/Redemption/Private/RedemptionPlayer.cpp

```diff
@@ -173,3 +222,25 @@ void ARedemptionPlayer::EndCrouch()
 
 	GetCapsuleComponent()->SetCapsuleSize(42.f, 96.0f);
 }
+
+void ARedemptionPlayer::StartSprinting()
+{
+	UE_LOG(LogTemp, Warning, TEXT("StartSprinting called"));
+	if (Stamina > 0 && bCanSprint)
+	{
+		bIsSprinting = true;
+		GetCharacterMovement()->MaxWalkSpeed = MaxSprintSpeed;
+	}
+}
+
+void ARedemptionPlayer::StopSprinting()
+{
+	UE_LOG(LogTemp, Warning, TEXT("StopSprinting called"));
+	bIsSprinting = false;
+	GetCharacterMovement()->MaxWalkSpeed = 500.f;
```

Comment:
![Screenshot 2024-03-02 153821](https://github.com/mac2648/Redemption/assets/60781347/f4a6a41a-9f3b-4081-8740-08ac14eaf545)
this is the player normal speed